10up/block-catalog
# Feature/upgrade php and phpunit

### Description of the Change

- Upgrade PHPUnit to 9.x
- Fixes failing tests on PHPUnit 9.x
- Fixes PHP warnings when running on PHP 8.2

Closes #33

### How to test the Change

- With the Block Catalog plugin active, on PHP 8.2+
- Run any Block Catalog or WP CLI command like, `wp block-catalog index`
- You should not see any PHP Warnings

### Changelog Entry

Fixed - PHP 8.2 deprecation warnings
Added - PHPUnit 9.x support

### Credits

Props @dsawardekar 

### Checklist:
